lms/templates/certificates/_accomplishment-rendering.html

```diff
@@ -51,7 +51,7 @@ <h3 class="accomplishment-signatories-title sr-only">${_("Noted by")}</h3>
 
                                     <h4 class="signatory-name hd-5">${signatory['name']}</h4>
                                     <p class="signatory-credentials copy copy-micro">
-                                        <span class="role">${signatory['title']|linebreaks}</span>
```

Comment:
@mattdrayer I found out that the linebreaks inserted a p with no class which messed up the styling, but I found a fix with css and put that in. Let me know if you think it's ok to remove the linebreaks import above.